### ISSUE TITLE
Add ET to the Data options

### ISSUE THREAD

**Issue by florianhartig** [biometry/RLPJ#10]
calculate ET as a sum of evaporation, transpiration, intercept

**Comment by Rasilgon** [comment]
I thought about this and my idea is to add a parameter _fun_. 

Fun could take any supported function - included in the package- as "met" as well as user defined functions. The function is applied with a try statement, so that invalid functions will not crash runLPJ. 
It is up to users to code properly their own function.

In this way, we keep a clear distinction between what is model outputs and package derived values, while we allow users to profit from the parallelization of LPJ-GUESS for processing the ouputs. We can show our supported functions in the manual and include examples in the vignette..

The other option will be to include it as part of typeList. In this case, the additional inputs will be provided as they were model outputs.

Either way, it will not affect the structure of the outputs. The additional variable are added to the LPJData object.

I have already tested the _fun_ option with met and an user defined function. To me seems a better option than using typeList.

**Comment by Rasilgon** [comment]
Should we include this in the package @florianhartig  @mauriziobagnara . I have it ready and tested.

Example:
At the  moment, if you want to calculate ET, you would  do something like this:

```
calculateET <- function(x){    
         x["met"]  <-  x["maet"] +  x["mevap"] + x["mintercep"]) 
         return(x)
 }
```

and then run it on the outputs

`LPJoutputs <-  lapply(LPJoutputs, calculateET)`

If there will be fun argument, one could call runLPJ as follows 

`LPJoutputs <- runLPJ(x=LPJsetup, parameterList=parameterMatrix,  settings=LPJsettings, fun = calculateET)`

This would give user the opportunity to do whatever they want to do with the LPJ outputs in parallel.
Internally, FUN is handled as 

```
LPJout <- applyFun(LPJout, fun)

applyFun <- function(x, fun){
  result <- try(fun(x), FALSE)
  if ('try-error' %in% class(result)){
    warning("Fun could not be applied")
    return(x)
  }else{
    return(result)
  }
```

So even if users defined wrong code, it will not crash the parallelization. Morevoer, it poses no problem when defining the typeList.
Let me know.

**Comment by mauriziobagnara** [comment]
can be included in #11
